Kickstarter-iOS/DataSources/BackerDashboardPagesDataSource.swift

```diff
@@ -0,0 +1,57 @@
+import KsApi
+import Library
+import UIKit
+
+internal final class BackerDashboardPagesDataSource: NSObject, UIPageViewControllerDataSource {
+  private let viewControllers: [UIViewController]
+
+  internal init(delegate: UIViewController, sort: DiscoveryParams.Sort) {
```

Comment:
I chose generic because the other 2 tabs will not be of this type (Following and Live Streams)